lib/recurly/fraud.js

```diff
@@ -62,22 +61,45 @@ export class Fraud extends Emitter {
     this.dataCollectorInitiated = true;
 
     this.recurly.request.get({
-      route: '/fraud_data_collector',
+      route: '/risk/info',
       done: (error, response) => {
-        debug('response from recurly data collector', error, response);
+        debug('risk info', error, response);
+
         if (error) {
-          const requestFailedError = errors('fraud-data-collector-request-failed', { error });
-          this.emit('error', requestFailedError);
-        } else if (response.content) {
-          const form = dom.element(this.recurly.config.fraud.kount.form) || this.getHostedFieldParentForm();
-          const tempContainer = document.createElement('div');
-          tempContainer.innerHTML = response.content;
-          each(tempContainer.childNodes, node => {
-            this.attachedNodes.push(node);
-            form.appendChild(node);
-          });
-          this.emit('ready');
+          return this.emit('error', errors('fraud-data-collector-request-failed', { error }));
         }
+
+        const { profiles } = response;
+
+        profiles.forEach(profile => {
+          const { processor, params } = profile;
+          if (processor === 'kount') {
+            const configuredForm = this.recurly.config.fraud.kount.form;
```

Comment:
Do we still need the fallback here in case a form isn't specified in the fraud config?